src/__tests__/Sessions.ts

```diff
@@ -231,4 +231,58 @@ describe("Sessions", () => {
     // Then
     expect(sessionTracker.sessionLengthMillis).toBe(sessionTimeMs);
   });
+
+  /*   test("Delays session init if items from last sessions are not loaded on android", () => {
+    // Given
+    const callMoment = Date.now();
+
+    storage.removeItem(AvoSessionTracker.lastSessionTimestampKey);
+
+    AvoInspector.avoStorage.storageInitialized = false;
+
+    // When
+    let sessionTracker = new AvoSessionTracker(mockBatcher);
+
+    // Then
+    expect(sessionTracker.lastSessionTimestamp).toBe(0);
+
+    // When
+    sessionTracker.startOrProlongSession(callMoment);
+
+    // Then
+    expect(sessionTracker.lastSessionTimestamp).toBe(0);
+
+    // When
+    AvoInspector.avoStorage.initializeStorageAndroid();
+
+    // Then
+    expect(sessionTracker.lastSessionTimestamp).toBe(callMoment);
+  }); */
```

Comment:
Do we want to put these back in place?

Reply:
The drawback of this implementation is that it's way harder to test other platforms in browser environment, so the next test is actually for web and I'm going to remove this one.